Tests/FoundationEssentialsTests/FileManager/FileManagerTests.swift

```diff
@@ -1046,6 +1046,85 @@ private struct FileManagerTests {
         }
     }
 
+    #if !os(Windows) && !os(WASI) && !os(OpenBSD) && !canImport(Android)
+    @Test func extendedAttributesDoNotFollowSymlinksWhenSetting() async throws {
+        let xattrKey = FileAttributeKey("NSFileExtendedAttributes")
+        #if canImport(Darwin)
+        let attrName = "com.swiftfoundation.symlinktest"
+        let probeName = "com.swiftfoundation.symlinkprobe"
+        #elseif os(Linux)
+        // Linux requires the user.* namespace prefix for regular files
+        let attrName = "user.swiftfoundation.symlinktest"
+        let probeName = "user.swiftfoundation.symlinkprobe"
+        #else
+        let attrName = "swiftfoundation.symlinktest"
+        let probeName = "swiftfoundation.symlinkprobe"
+        #endif
+        let attrValue = Data([0xAA, 0xBB, 0xCC])
+        let probeValue = Data([0x11, 0x22, 0x33])
+
+        try await FilePlayground {
+            File("target", contents: Data("payload".utf8))
+            SymbolicLink("link", destination: "target")
+        }.test { fileManager in
+            // First, prove that this environment supports xattrs on regular files and that we
+            // have permission to set them. If this fails, the symlink behavior isn't meaningful.
+            do {
+                try fileManager.setAttributes([xattrKey: [probeName: probeValue]], ofItemAtPath: "target")
+            } catch let error as CocoaError {
+                if error.code == .featureUnsupported { return }
+                guard let posix = error.underlying as? POSIXError else { throw error }
+                #if canImport(Darwin)
```

Comment:
On the `ENOTSUP` vs. `EOPNOTSUPP` here, if we're just using the raw values instead I think you can switch this to just

```swift
posix.code.rawValue == EOPNOTSUPP || posix.code.rawValue == ENOTSUP
```

for all platforms and call it good since the raw values exist on both platforms, which would simplify this code a lot. Is that correct?

Reply:
Yeah